src/Model.cxx

```diff
@@ -313,13 +321,17 @@ void Model::addDataAccessor(DataAccessorSet::value_type da)
         // adding to StaticDataAccessorVector insures that
         // HelicityAngles are called before any newly created
         // StaticDataAccessors
-        if (!HelicityAngles_ and dynamic_cast<RequiresHelicityAngles*>(da))
+        if (!HelicityAngles_ and dynamic_cast<RequiresHelicityAngles*>(da)) {
```

Comment:
We should add functionality to `RequiresHelicityAngles` like we have for `requiresMeasuredBreakupMomenta` so as to accommodate when something inherits from it but doesn't actually need the helicity angles (aka when dealing with spin zero).